tests/unit/mixins/Registry.ts

```diff
@@ -57,7 +57,7 @@ registerSuite({
 			class IntegrationTest extends TestWithRegistry {
 				render() {
 					return v('div', [
-						w('test', { id: `${Math.random()}` })
+						w<any>('test', { id: `${Math.random()}` })
```

Comment:
Have we run this against `todo-mvc` or `todo-mvc-kitchensink` to see if we get any compile errors (wanted or unwanted)?  Because an alternative is to have an overload with a 'string' that would not require a supplied generic like this.  I guess my feeling is that `<any>` seems like an odd generic to have to supply.  It is like telling TypeScript to ignore something it can reasonably ignore anyways and lends end developers to just use the `<any>` hammer everywhere to just keep TypeScript quiet, whether they are using the registry or not.